src/SearchResults/SearchResults.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import moment from "moment";
+
+import "./SearchResults.css";
+
+const SearchResults = props => {
+  return (
+    <table className="table">
+      <thead>
+        <tr>
+          <th>Title</th>
+          <th>First Name</th>
+          <th>Surname</th>
+          <th>Email</th>
+          <th>Room id</th>
+          <th>Check in date</th>
+          <th>Check out date</th>
+          <th>Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map(booking => {
+          let {
+            id,
+            title,
+            firstName,
+            surname,
+            email,
+            roomId,
+            checkInDate,
+            checkOutDate
+          } = booking;
```

Comment:
this is fine but remember we can also destructure inside the method signature 
```suggestion
        {props.results.map(({
            id,
            title,
            firstName,
            surname,
            email,
            roomId,
            checkInDate,
            checkOutDate
          }) => {
```